lib/govuk_sidekiq/sidekiq_initializer.rb

```diff
@@ -15,7 +16,7 @@ def self.setup_sidekiq(govuk_app_name, redis_config)
         config.redis = redis_config
 
         config.server_middleware do |chain|
-          chain.add Sidekiq::Statsd::ServerMiddleware, env: "govuk.app.#{govuk_app_name}", prefix: "workers"
+          chain.add Sidekiq::Statsd::ServerMiddleware, statsd: GovukStatsd, env: nil, prefix: "workers"
```

Comment:
This looks really good. Following on from the statsd gauge related changes, I was looking at the sidekiq statsd metrics, and was thinking that there'll be some contention between different machines, which this should resolve 👍